datalad_dataverse/create_sibling_dataverse.py

```diff
@@ -68,10 +68,29 @@ class CreateSiblingDataverse(Interface):
     Dataverse is a web application to share and cite research data.
     Research data published in Dataverse receives an academic citation which allows to grant full credit and increases visibility of your work.
     
+    In order to be able to use this command, a personal access token has to be generated on the Dataverse platform. You can find it by clicking on your name at the top right corner, and then clicking on Api Token>Create Token.
+
     TODO: command doc
     """
 
-    _examples_ = []
+    _examples_ = [
+        dict(text="Create a dataset sibling in the form of a dataverse dataset",
+             code_py="""\
+                 > ds = Dataset(url='https://demo.dataverse.org')
```

Comment:
Sorry, I didn't mean to suggest that this part should be changed :) Lets keep it as you previously had it:

```suggestion
                 > ds = Dataset('.')
```